src/handlers/comment/handlers/issue/identify-user-ids.ts

```diff
@@ -31,7 +31,12 @@ async function filterUsers(context: Context, issue: Issue, contributorComments:
     (user: User) => !allRoleUsers.some((_user) => _user?.id === user.id)
   );
   const uniqueContributors = Array.from(new Map(contributors.map((user) => [user.id, user])).values());
-  return { issuer, assignees, collaborators, contributors: uniqueContributors };
+  return {
+    issuer,
+    assignees,
+    collaborators: collaborators.filter((collaborator) => collaborator.id !== issuer.id),
```

Comment:
Seems like kind of a hack instead of solving the root problem, but I suppose we can refine later.

Reply:
I think it can be a root solution.
Cause the it was only happening on the issue author's comment.
So we can just remove the issue author from collaborators list.